source/CGameMenu.cpp

```diff
@@ -24,6 +24,8 @@ namespace CLEO
 
     void __fastcall OnDrawMenuBackground(void *texture, int dummy, RwRect2D *rect, RwRGBA *color)
     {
+        GetInstance().Start(true); // late initialization
```

Comment:
why is it called from here and not from OnScmInit1 for example?

Reply:
That would be very late. If level 3 of logging is enabled there are already messages to display in main menu.
Future ambient scripts will also log stuff right after game starts.